src/modm/platform/spi/bitbang/bitbang_spi_master_impl.hpp

```diff
@@ -48,20 +45,23 @@ modm::platform::BitBangSpiMaster<Sck, Mosi, Miso>::connect()
 	static constexpr bool ContainsMosi = Connector::template Contains<Mosi>;
 	static constexpr bool ContainsMiso = Connector::template Contains<Miso>;
 	static_assert(ContainsSck and ContainsMosi and
-				  ((not Connector::template IsValid<Miso> and sizeof...(Signals) == 2) or
-				   (    Connector::template IsValid<Miso> and sizeof...(Signals) == 3 and ContainsMiso)),
-				  "BitBangSpiMaster<Sck, Mosi, Miso> can only connect to the same Sck, Mosi and Miso signals of the declaration!");
+					  ((not Connector::template IsValid<Miso> and
+						sizeof...(Signals) == 2) or
+					   (Connector::template IsValid<Miso> and
+						sizeof...(Signals) == 3 and ContainsMiso)),
+				  "BitBangSpiMaster<Sck, Mosi, Miso> can only connect to the "
+				  "same Sck, Mosi and Miso signals of the declaration!");
```

Comment:
How in hell is this more readable?

Reply:
I believe this is caused by indenting the line wrap 4 tabs (should only be one). Let me fix that...